.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Unit tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.4.2'
```

Comment:
can we use latest instead of a specific version?

Reply:
I think the best approach would be to mirror the `.nvmrc` version. But I don't know how to get the version from that file in the CI machine. What do you think? :thinking: We can use the latest version, but it could be a different version from the version we use in the project.

Reply:
that would be really great if we could, but I think it's tough to pull that info and probably not worth the syncinc effort. 
You're right in that use latest could risk the versions falling out of sync....however, we plan to keep this project up to date with, at a minimum, major versions of node so there shouldn't be a problem.

Reply:
Great, I'll change to the latest version! :)

Reply:
@ermish, I'm trying to find ways to use the latest version, but it seems that we can only use "fixed" versions. Take a look at [this syntax](https://github.com/actions/setup-node#supported-version-syntax).

We can use like this `16` or more specific like `16.4.2`.

Or we can use the `lts`, but I think not the latest version.

- `lts/erbium`: v12
- `lts/fermium`: v14

But it seems that it doesn't have for v16 for example :thinking: Do you have any advice here?

- [resource](https://nodejs.org/en/about/releases/).

Reply:
Made it work using nvm https://github.com/ourjapanlife/findadoc-frontend/pull/287/commits/a1207cc3fb62bedf5c52b010423f819f3e880d08#diff-cd0effce06f425599e96952f4d5f684641a683f46d9909e7c74a9f8b2f03ccd3R16 :tada:

Reply:
ha, nicely done! way to figure it out 🥇